README.md

```diff
@@ -140,6 +140,17 @@ To run the tests:
 > vcbuild test
 ```
 
+### Android / Android based devices, aka. Firefox OS
+
+Be sure you have downloaded and extracted [Android NDK]
+(https://developer.android.com/tools/sdk/ndk/index.html) 
+before in a folder. Then run:
+
+```sh
```

Comment:
can you remove the `sh` please, we're showing these shell examples without that formatting because it makes little sense and isn't consistent when you're not showing bash anyway (plus, `$` isn't a bashism as it's used here)